timesketch/frontend-ng/src/components/Scenarios/Question.vue

```diff
@@ -101,20 +124,22 @@ limitations under the License.
 
 <script>
 import ApiClient from '../../utils/RestApiClient'
-import TsSearchTemplate from '../LeftPanel/SearchTemplateCompact'
+import TsSearchChip from './SearchChip'
 import TsQuestionConclusion from './QuestionConclusion'
 
 export default {
   props: ['question'],
   components: {
-    TsSearchTemplate,
+    TsSearchChip,
     TsQuestionConclusion,
   },
   data: function () {
     return {
       expanded: false,
       fullDescription: false,
       conclusionText: '',
+      addConclusion: false,
+      opensearchQueries: [],
```

Comment:
General comment for this PR, in other places (like https://github.com/google/timesketch/blob/master/timesketch/frontend-ng/src/components/LeftPanel/SearchTemplate.vue#L103) we are using `queryString`. So maybe we could use something similar here as well.

Reply:
I want to be explicit on what this is. We have queries coming from both searchtemplates and from "raw" opensearch queries.